src/app/type.d.ts

```diff
@@ -0,0 +1,5 @@
+declare module '*.svg' {
```

Comment:
We already added this to https://github.com/joshsoftware/intranet-mobile/blob/develop/src/types/index.d.ts

Reply:
So, do we need to create this again?